### DISCUSSION TITLE
Custom Client vs AWS Hubs Cloud

### DISCUSSION THREAD

**Discussion by alfainfotech** [mozilla/hubs#4985] [Hubs Cloud Questions] [answered]
Hello experts,

I was able to install AWS Hubs cloud and running fine, but in order to customize (eg: homepage design modification, adding own custom links etc., ) how can do I AWS Hubs cloud ? I even tried to install FTP but was not able to connect looks like FTP port is closed in AWS config. Before doing any modification wanna know 1st if AWS Hubs cloud can be customized ? If so how can we access the files. 

Appreciate any help

**Comment by matthewbcool** [comment, marked as answer]
The client is very customizable.  

**Examples of custom clients:**
- [Scavenger Hunt by Mannmade.co.za](https://gyazo.com/258518d6d4a74aab6f9640eecdbd4209) 
- [imedu.io](https://youtu.be/zSO43wOgbVA) 
- [estraad.com](https://youtu.be/ZSOeokFGiS0) 
- [Apart Posters by Paradowski](https://paradowski.com/work/apart) 

Some minor customization options are available to you through the admin panel but for greater control you will need to deploy your own code.  You can make changes to the [client re:how to access files](https://github.com/mozilla/hubs) and test those changes either locally or against your hubs cloud and deploy your client from there.

More info here: https://hubs.mozilla.com/docs/hubs-cloud-customizing-look-and-feel.html 

Search Github discussion and #hubs-cloud in discord for additional info.

**Reply by alfainfotech** [reply]
Thanks mattewbcool 
Yes I understand working locally on customization, the part Im trying to learn is how to deploy your custom code into AWS Hubs Cloud.  This would help. Or if it possible to FTP into AWS.

**Reply by matthewbcool** [reply]
"test those changes either locally or against your hubs cloud and deploy your client from there."

more info on the deploying part of that here:
https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html
